extension/api.js

```diff
@@ -83,10 +88,20 @@
 			url += 'api/v3/notifications';
 		}
 
-		url += participating;
+		url += `?${query.join('&')}`;
 
 		xhr('GET', url, opts, (data, status, response) => {
```

Comment:
You need to perform additional requests only if last notification timestamp is changed. The plugin can set flag to fetch full first page on next API call if that desktop notification is needed. I have 2054 pending notifications and I am not a user for desktop notifications.

It also seems to me that polling API is not effective for desktop notifications stuff. I'd ask GitHub is they have some pubsub API for real-time notifications. For me it seems even more effective than hammering the server every minute.
https://developer.github.com/v3/activity/events/#list-events-that-a-user-has-received